docs/architecture/devops-for-aspnet-developers/actions-build.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Deploy an app to App Service - DevOps with .NET and GitHub Actions
+description: Deploy a .NET app to Azure App Service, the first step for DevOps with .NET and GitHub Actions.
+author: colindembovsky
+ms.date: 03/04/2021
+---
+# Deploy an app to App Service - DevOps with .NET and GitHub Actions
+
+[GitHub Actions](https://github.com/features/actions) allow you to automate workflows in response to events that are triggered in GitHub. Common workflows include Continuous Integration (CI) workflows, but Actions can be used to automate other processes such as sending out welcome emails when people join a repositorysitory.
+
+To explore how you can move code to the cloud, you'll build a GitHub Actions workflow file for the Simple Feed Reader app you've already deployed to Azure App Service.
+
+In this article, you will:
+> [!div class="checklist"]
+
+> * Learn the basic structure of a GitHub Action workflow YAML file.
+> * Use a template to create a basic build workflow that builds the .NET app and executes unit tests.
+> * Publish the compiled app so that it's ready for deployment.
+
+## Workflow structure
+
+Workflows are captured in YAML files and all have the following things:
+
+- a `name`
+- a trigger, defined by an `on` section
+- one or more `job` sections, composed of one or more `steps`
+- optional attributes such as `environment` variables
+
+Jobs are run on _runners_. You can use _hosted runners_, which are spun up by GitHub during the workflow and then thrown away. Hosted runners are great because you don't have to maintain your own build infrastructure. For workflows that require a specific build environment, or for running workflows on a private network, you can also use _private_ runners. To create a private runner, you can install the runner on any machine that supports .NET.
```

Comment:
```suggestion
Jobs are run on _runners_. You can use _hosted runners_, which are spun up by GitHub during the workflow and then thrown away. Hosted runners are great because you don't have to maintain your own build infrastructure. For workflows that require a specific build environment, or for running workflows on a private network, you can also use _private_ runners. To create a private runner, install the runner on any machine that supports .NET.
```